Dart/resources/messages/DartBundle.properties

```diff
@@ -22,6 +22,7 @@ dart.color.settings.description.symbol.literal=Symbol literal
 dart.color.settings.description.annotation=Annotation
 dart.color.settings.description.class=Class
 dart.color.settings.description.constructor=Constructor
+dart.color.settings.description.constructor.tearoff=Constructor tear off
```

Comment:
Looking through the [spec](https://github.com/dart-lang/language/blob/master/accepted/future-releases/constructor-tearoffs/feature-specification.md) I noticed that noun is usually spelled as `tear-off`, and verb as `tear off`. Should it be `tear-off` in color settings?

Reply:
I went back and forth, but don't have a strong preference, changing to `tear-off`